tests/test_timefuncs.py

```diff
@@ -534,3 +534,104 @@ def test_types_to_numpy() -> None:
     check(assert_type(td_s.to_numpy(), np.ndarray), np.ndarray)
     check(assert_type(td_s.to_numpy(dtype="int", copy=True), np.ndarray), np.ndarray)
     check(assert_type(td_s.to_numpy(na_value=0), np.ndarray), np.ndarray)
+
+
+def test_to_timdelta_units() -> None:
+    pd.to_timedelta(1, "W")
+    pd.to_timedelta(1, "w")
```

Comment:
For consistency, should do the `check(assert_type(` pattern here.

Reply:
These have been added